pkg/controller/workload/workload_processor.go

```diff
@@ -275,13 +278,20 @@ func (p *Processor) removeServiceResourceFromBpfMap(svc *workloadapi.Service, na
 		}
 
 		var i uint32
-		for i = 1; i <= svDelete.EndpointCount; i++ {
-			ekDelete := bpf.EndpointKey{
-				ServiceId:    serviceId,
-				BackendIndex: i,
+		var j uint32
```

Comment:
nit: you donot need to define j here, move below

for j:=0; j <= bpf.MaxPrio; j++ {

Reply:
ok